app/main.ipynb

```diff
@@ -493,26 +1682,61 @@
     "    result = total / len(lst)\n",
     "    return result\n",
     "\n",
-    "lst = [1, 2, 3, 4, 5]"
+    "lst = [1, 2, 3, 4, 5]\n",
+    "unknown_signature(lst)"
```

Comment:
Ensure the result of the `unknown_signature` function is printed to verify correctness.